ray_on_golem/server/settings.py

```diff
@@ -122,3 +41,111 @@
 PAYMENT_NETWORK_POLYGON = "polygon"
 PAYMENT_NETWORK_GOERLI = "goerli"
 PAYMENT_DRIVER_ERC20 = "erc20"
+
+RAY_ON_GOLEM_PATH = Path(os.getenv("RAY_ON_GOLEM_PATH", "ray-on-golem"))
+YAGNA_PATH = Path(os.getenv("YAGNA_PATH", "yagna"))
+WEBSOCAT_PATH = Path(os.getenv("WEBSOCAT_PATH", "websocat"))
+TMP_PATH = Path("/tmp/ray_on_golem")
```

Comment:
Do we still need TMP_PATH

Reply:
we use it for the location of the ssh key pair (on the provider machines)